### DISCUSSION TITLE
Combining prediction scores from different references

### DISCUSSION THREAD

**Discussion by danielcgingerich** [stuart-lab/signac#576] [Q&A] [answered]
I will first try to explain the experimental setup:

There are basically two huge single cell data sets - 1 for ATAC seq, and 1 for RNA seq.  Each of these data sets were collected from the same 24 samples, same brain region.  The scRNA data set was annotated using a reference from Allen Brain Institute, found [here](https://portal.brain-map.org/atlases-and-data/rnaseq/human-m1-10x).  The scATAC data set was annotated using the same reference from Allen Institute.  I also annotated the scATAC data using our own scRNA data as a reference.  The predictions between our reference versus the Allen reference were about 75% consistent.  Of the cells that did not match, prediction scores found from our homemade scRNA reference were much higher than the Allen reference.  

Here is what I am wondering: 
How do I combine the prediction scores from the two different references used?  Is there a more sophisticated way than just picking the highest `prediction.score.max` from the two references? 

Why is there so much inconsistency between the two different references?  I think its weird because our own scRNA reference was annotated with the Allen reference to begin with, yet the cell type predictions from the scRNA reference do not match with the Allen reference.

**Comment by timoast** [comment, marked as answer]
If the cell labels are the same across references that you're mapping to, you could try taking the highest prediction score across both mappings, or maybe combining the scores by taking a product and then taking the max (which will require the score was high for both mappings). We have not explored this, so you should interpret with caution and always look at the original data to see if the predicted labels make sense. Another option would be to integrate your scRNA reference and the Allen reference to create a single new reference dataset, and map the scATAC data to that.
